geps/gep-2643/index.md

```diff
@@ -0,0 +1,439 @@
+
+# GEP-2643: TLS/SNI based routing / TLSRoute
+
+* Issue: [#2643](https://github.com/kubernetes-sigs/gateway-api/issues/2643)
+* Status: Standard
+
+## TLDR
+
+This GEP documents the implementation of a route type that uses the Server Name
+Indication attribute (aka SNI) of a TLS handshake to make the routing decision.
+
+While this feature is also known sometimes as TLS passthrough, where after the 
+server name is identified, the gateway does a full encrypted passthrough of the 
+communication, this GEP will also cover cases where a TLS communication is 
+terminated on the Gateway before being passed unencrypted to a backend.
```

Comment:
```suggestion
terminated on the Gateway before being passed to a backend.
```

"unencrypted" is unnecessary as there are options for re-encrypting the Gateway auto Backend. The “Request flow” covers both cases well.